tests/solana.rs

```diff
@@ -1757,3 +1600,149 @@ pub fn parse_and_resolve(src: &'static str, target: Target) -> ast::Namespace {
 
     solang::parse_and_resolve(OsStr::new("test.sol"), &mut cache, target)
 }
+
+struct VmFunction<'a, 'b> {
+    vm: &'a mut VirtualMachine,
+    idx: usize,
+    _expected: u64,
+    _accounts: Vec<AccountMeta>,
+    has_remaining: bool,
+    _arguments: Option<&'b [BorshToken]>,
```

Comment:
Why  the ugly underscores?

Reply:
Why is this error prone? Rust allows struct members and functions to have the same name.

Reply:
> Why is this error prone? Rust allows struct members and functions to have the same name.

If you do `vm.function("new").accounts` the code compiles without error, but that is not what you want.

Reply:
I've removed the underscores.